host/src/operations/mod.rs

```diff
@@ -313,16 +337,4 @@ pub fn execute<T: serde::Serialize + ?Sized, O: Eq + Debug + DeserializeOwned>(
             output_guest, expected_output,
         );
     }
-    // report performance
-    println!(
-        "Generated {:?} segments; elapsed time: {:?}",
-        session.segments.len(),
-        start_time.elapsed()
-    );
-    println!(
```

Comment:
why'd we lose the timer?

Reply:
The new risc0 logs the execution time with `info`. Do you think the println is still needed?

Reply:
```
[2024-02-09T16:41:59Z INFO  risc0_zkvm::host::server::exec::executor] execution time: 1:02:55
[2024-02-09T16:41:59Z INFO  risc0_zkvm::host::server::session] number of segments: 34837
[2024-02-09T16:41:59Z INFO  risc0_zkvm::host::server::session] total cycles: 36528717824
[2024-02-09T16:41:59Z INFO  risc0_zkvm::host::server::session] user cycles: 33155587005
[2024-02-09T16:41:59Z INFO  risc0_zkvm::host::server::session] cycle efficiency: 90%
```